postpic/datareader/epochsdf.py

```diff
@@ -36,10 +36,24 @@
 import numpy as np
 import re
 from .. import helper
+from warnings import warn
 
 __all__ = ['Sdfreader', 'Visitreader']
 
 
+# The default staggering of the Grid.
+# Newer versions of EPOCH dump it (certainly v4.9.2)
+# older versions dont. The defaults will only be used,
+# if the stagger was not found in the dump.
+_default_stagger = {'Electric Field/Ex': 1,
+                    'Electric Field/Ey': 2,
+                    'Electric Field/Ez': 4,
+                    'Electric Field/Bx': 6,
+                    'Electric Field/By': 5,
+                    'Electric Field/Bz': 3}
+_default_stagger.update({k+'_averaged': v for k, v in _default_stagger.items()})
+
```

Comment:
We should make sure that we add all default_staggers that are available for epoch. jx/jy/jz are definitely also available. There could be even more cases, we should grep Epoch for stagger.